akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -165,7 +165,8 @@ private[http2] class Http2ServerDemux(http2Settings: Http2ServerSettings, initia
                 multiplexer.pushControlFrame(SettingsAckFrame(settings))
               }
 
-            case PingFrame(true, _) =>
+            case SettingsAckFrame(Nil) => // This should be the ack to the initial settings frame
```

Comment:
So we don't care and just drop the ack? Seems a bit suspicious but :+1:

Reply:
It was just logged as unhandled before (debug level).

Reply:
Yes, in that sense the behavior doesn't really change, but handling it explicitly like this gives the impression that it is 'correct' to just ignore the Ack, which was surprising to me - you'd expect it to be in the protocol for a reason.

Reply:
Yes, exactly, and it's an answer to the frame we send in `preStart`. Should we start sending proper things in there, we need to revise.

Reply:
for now perhaps add a comment here explaining what we should ideally do (even though we don't do it now)?

Reply:
Good point, added https://github.com/akka/akka-http/issues/3185 and linked to it from the comment.